nvtabular/torch_dataloader.py

```diff
@@ -218,20 +222,75 @@ def __len__(self):
         return self.num_chunks
 
     def __iter__(self):
-        buff = queue.Queue(1)
-        threading.Thread(target=self.load_chunk, args=(buff,)).start()
-        for _ in range(self.num_chunks):
-            chunk = buff.get()
-            yield from TensorItr(
-                chunk, batch_size=self.batch_size,
-            )
-
-    def load_chunk(self, out):
         for chunk in self.itr:
-            chunk = create_tensors_plain(
-                chunk, cat_cols=self.cat_cols, cont_cols=self.cont_cols, label_cols=self.label_cols
-            )
-            out.put(chunk)
+            yield chunk
+
+
+class ChunkQueue:
```

Comment:
Documentation here would help a lot, especially to describe what the difference between `num_chunks` and `batch_size` is.

Reply:
Its been changed to batch_size